### ISSUE TITLE
Pagerduty gateway - don't check for acks if subdomain, username and password are invalid

### ISSUE THREAD

**Issue by jessereynolds** [flapjack/flapjack#451]
If a contact's pagerduty media does not have a valid subdomain, username and password, then don't attempt to look for acknowledgements on pagerduty for this contact. 

Eg, currently if subdomain is empty, the pagerduty gateway will try and make API requests to pagerduty like:

https://.pagerduty.com/api/v1/incidents

which isn't even a valid URL.

**Comment by chrisgagne** [comment]
I don't believe the username, password, or subdomain are necessary anymore. This may have been a legacy bit of code.
